src/Components/Operators.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import CSS from "csstype";
+import "../App.css";
+
+interface Props {
+  handleClick(el: string): void;
+  handleSubmit(): void;
+}
+
+const Operators = ({ handleClick, handleSubmit }: Props) => {
+  const operators = ["+", "-", "/", "*", "="];
+  const operatorComponents = operators.map(
+    (operator: string, index: number) => {
+      const onClick = operator === "=" ? handleSubmit : () => handleClick(operator);
```

Comment:
This component feels nearly identical to numbers. I think you could probably get rid of both components entirely and move all the logic to the calculator. When a button is pressed, it could just tell Calculator its value and calculator can handle the display change.